test/puppetlabs/comidi_test.clj

```diff
@@ -67,10 +37,6 @@
       (is (= {:path           ["/foo"]
               :request-method :any}
              (update-route-info* orig-route-info "/foo"))))
-    (testing "keyword path elements get added to the path"
-      (is (= {:path           [:foo]
-              :request-method :any}
-             (update-route-info* orig-route-info :foo))))
```

Comment:
Good call, I removed it because it failed schema validation but a slightly more careful look shows that the keyword needs to be in a vector. Does that seem alright?